clang/bindings/python/README.txt

```diff
@@ -4,6 +4,8 @@
 
 This directory implements Python bindings for Clang.
 
+The bindings library requires at least Python version 3.7.
```

Comment:
Let's drop this change for now and keep the testing at 3.7. We can add this in later if the libclang bindings have a minimum version explicitly higher than the rest of the project, or we won't need it if the minimum for the entire project gets raised.

Reply:
Agree.

Reply:
Fair. I dropped this change,